src/routers/datatypeRouters/edges/variants_phenotypes.ts

```diff
@@ -95,21 +110,36 @@ async function findVariantsFromPhenotypesSearch (input: paramsFormatType): Promi
 
   if (input.phenotype_id !== undefined) {
     query = `
-    LET primaryEdge = (
-        For record IN ${variantToPhenotypeSchema.db_collection_name as string}
-        FILTER record._to == 'ontology_terms/${input.phenotype_id as string}'
-        RETURN record._id
-    )
-
-    FOR edgeRecord IN ${variantPhenotypeToStudy.db_collection_name as string}
-    FILTER edgeRecord._from IN primaryEdge ${pvalueFilter}
-    SORT edgeRecord._key
-    LIMIT ${input.page as number * limit}, ${limit}
-    RETURN {
-      'study': ${input.verbose === 'true' ? `(${verboseQuery})[0]` : 'edgeRecord._to'},
-      ${getDBReturnStatements(variantPhenotypeToStudy).replaceAll('record', 'edgeRecord')},
-      'name': edgeRecord.inverse_name // endpoint is opposite to ArangoDB collection name
-    }
+      FOR record IN ${variantToPhenotypeSchema.db_collection_name as string}
+      FILTER record._to == 'ontology_terms/${input.phenotype_id as string}' ${filesetFilter}
+
+      LET igvf = (
+        SORT record._key
```

Comment:
Missing for loop?

Reply:
It's within the FOR loop from line 113. It looks strange, but it saves the proper records into the `igvf` variable correctly